src/lox.py

```diff
@@ -17,10 +17,14 @@ def run(self, source):
 
     # run file directly
     def runFile(self, path):
-        # TODO: If the file doesn't exist
+        # If the file doesn't exist
         # exit gracefully
-        with open(path) as f:
-            script = f.read()
+        try:
+            with open(path) as f:
+                script = f.read()
+        except IOError:
+            print("Sorry, the file doesn't exist!")
```

Comment:
We don't need to say `Sorry` :sweat_smile: . It's better to specify the file name in the error message. Hope this helps :)

Reply:
Okay fixing that.. :)